### ISSUE TITLE
DMSO images as additional negative examples

### ISSUE THREAD

**Issue by agitter** [gitter-lab/pharmaco-image#8]
When training image-based models to compare with the results in #7, we can also use the DMSO images.  These provide additional negative examples for all of the assays.  We would only use them to augment the training data.  The validation and test set data should remain the same so that the results are more comparable to models that train and evaluate only on the chemical features (e.g. fingerprints).

This would be applicable if we use pre-trained models to extract features or directly train the models described in #7.
